stylegan_code_finder/lightning_modules/base_lightning.py

```diff
@@ -0,0 +1,61 @@
+import pytorch_lightning as pl

```

Comment:
In the end you should also remove all the files you want to replace (train_builder, old train file, etc.)